.idea/compiler.xml

```diff
@@ -2,6 +2,6 @@
 <project version="4">
   <component name="TypeScriptCompiler">
     <option name="recompileOnChanges" value="true" />
-    <option name="typeScriptCompilerParams" value="--noEmit" />
+    <option name="typeScriptCompilerParams" value="--noEmit --target es5 --jsx react" />
```

Comment:
I needed to add this for it to build in WebStorm, so if nothing else from the .idea changes make it in, this should.

Reply:
This is an exception to my previous comment about changing the `.idea` files.